app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  def index
+    @tickets = Ticket.all
+    render json: @tickets
+  end
+
+  def create
+    reservation = [] 
+    
+    @ticket = Ticket.new(ticket_params).each do 
+      if @ticket.save
+        reservation << @ticket
+      else
+        render json: @tickets.errors, status: :unprocessable_entity
+      end
+    end
+    render json: reservation, status: :created
```

Comment:
How does this logic allow users to buy more than one ticket? Could you give examples of parameters that have to be sent here? I mean - how `ticket_params` should looks like?

Reply:
@wmaciejak  Tickets have an array "reservation" and after creating each of tickets, they are in this array. So in ticket_params I have to add price. What is more: type of ticket, movie id, screening id and seats id

Reply:
Could add only this additional `seats_id`? Generally looks, fine, only wanted to confirm your way of thinking :)